pkg/apis/core/validation/validation.go

```diff
@@ -1681,7 +1684,7 @@ func ValidationOptionsForPersistentVolume(pv, oldPv *core.PersistentVolume) Pers
 	return opts
```

Comment:
In general, once a field is set, we want to continue to validate the field so that it remains valid even if the feature gate is off.